### PR TITLE
feat: save bahis version at login only

### PR DESCRIPTION
**Pull request by shaarful**
## Description
Now the bahis version only will save at log in time. Instanceid column has been removed.

If the bahis desk version number is passed at log in time then save it in the table. For older bahis desk (2.2.1 and backward) application will not send the version number so save None in the table

- [x] I have read the road86 Contribution Guide.
- [x] I have checked all commit message styles match the requested structure.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have assigned at least one reviewer.
- [x] New and existing unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] My changes generate no new warnings.
- [x] I have made corresponding changes to the documentation.
- [x] New database changes have been committed.

### CODE REVIEW

File: kobocat/onadata/apps/bh_module/views_forms.py [hunk 1]
```diff
@@ -646,6 +647,9 @@ def submission_request(request, username):
 
         instance_data = instance_data.replace('uuid:', '')
         instance = Instance.objects.filter(uuid=instance_data).first()
+        # user = User.objects.get(username = username)
```
Comment (ChasNelson1990) [inline kobocat/onadata/apps/bh_module/views_forms.py]: Can we delete these please - we don't need more dead code lol

File: kobocat/onadata/apps/usermodule/models.py [hunk 2]
```diff
@@ -118,7 +118,7 @@ def __str__(self):
 
 #branch
 class Branch(models.Model):
-    id = models.BigIntegerField(null=False)
+    # id = models.BigIntegerField(null=False)
```
Comment (ChasNelson1990) [inline kobocat/onadata/apps/usermodule/models.py]: Can we delete this please - we don't need more dead code lol

**Review by ChasNelson1990 — CHANGES_REQUESTED** [review]
LGTM - just some dead code to remove

File: kobocat/onadata/apps/bh_module/views_forms.py (hunk 1 above)
Comment (shaarful) [reply kobocat/onadata/apps/bh_module/views_forms.py]: yes we can

**Comment by shaarful** [comment]
@ChasNelson1990 is it a good idea to save the Mac address also?

**Comment by shaarful** [comment]
Because we are getting Mac address from client request at login

**Comment by ChasNelson1990** [comment]
> @ChasNelson1990 is it a good idea to save the Mac address also?

So a lot of the codebase is set up to do that... but it was all commented out or set to null when I started working on this project. So i presume there were issues or a reason. As we don't need it right now I suggest ignoring it still?

**Comment by shaarful** [comment]
> > @ChasNelson1990 is it a good idea to save the Mac address also?
> 
> So a lot of the codebase is set up to do that... but it was all commented out or set to null when I started working on this project. So i presume there were issues or a reason. As we don't need it right now I suggest ignoring it still?

agreed, let's forget about Mac now.

**Review by ChasNelson1990 — COMMENTED** [review]
@sharful09 - wanna resolve the conflicts on this before I review it?

**Comment by shaarful** [comment]
> @sharful09 - wanna resolve the conflicts on this before I review it?

Resolved the conflicts

**Review by ChasNelson1990 — APPROVED** [review]

**Pull request merged**